requirements.txt

```diff
@@ -1,4 +1,4 @@
 beautifulsoup4==4.9.1
-Django==3.2.5
+Django==4.0.2
```

Comment:
This is the latest version according to https://pypi.org/project/Django/#history